### PR TITLE
Passing original flags

### PR DESCRIPTION
**Pull request by chienwei-lan**

### CODE REVIEW

**Comment by stsoe** [comment]
@chvamshi-xilinx .  Can you please make sure xclGetBOProperties returns the original flags also for emulation? 
@larry9523.  Ditto for zocl .
Thanks!

**Review by larry9523 — APPROVED** [review]

**Comment by larry9523** [comment]
> @chvamshi-xilinx . Can you please make sure xclGetBOProperties returns the original flags also for emulation?
> @larry9523. Ditto for zocl .
> Thanks!

Sure @stsoe , I will add it to zocl shortly.

**Comment by larry9523** [comment]
You may want to change the comment before drm_xocl_info_bo definition. @flags is no longer unused.

File: src/runtime_src/core/pcie/driver/linux/include/xocl_ioctl.h [hunk 1]
```diff
@@ -311,7 +311,7 @@ struct drm_xocl_sync_bo_cb {
  * used with DRM_IOCTL_XOCL_INFO_BO IOCTL
  *
  * @handle:	bo handle
- * @flags:      Unused
+ * @flags:      flags passed while create bo
```
Comment (larry9523) [inline src/runtime_src/core/pcie/driver/linux/include/xocl_ioctl.h:314]: 1) Indent
2) How about Flags passed while creating BO (out) 
:)

**Review by chvamshi-xilinx — APPROVED** [review]
Sure @stsoe . @venkatp-xilinx , Please make changes in emulation driver to return original flags.

**Pull request merged**